src/partials/config.html

```diff
@@ -98,6 +98,15 @@ <h3 class="page-heading">Connection details</h3>
 
 <h3 class="page-heading">Query Optimizations</h3>
 <div class ="gf-form-group">
+  <div class="gf-form-inline">
+    <div class="gf-form">
+      <span class="gf-form-label width-11">Query timeout</span>
```

Comment:
@mckn Min 1s, Max is 1h, can you adjust the UI to match that?

Reply:
Yupp - I fix :)